### PR TITLE
3 sign

### PR DESCRIPTION
**Pull request by sgreenbury**
Completes #3 and #36.

### CODE REVIEW

**Comment by sgreenbury** [comment]
9/16 files reviewed.

**Comment by lukehare** [comment]
11/16 files reviewed.

File: trustchain-ion/src/create.rs [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+        let signing_key = Some(generate_key());
+        let public_key_entry = PublicKeyEntry::try_from(signing_key.clone().unwrap());
+        (
+            DocumentState {
+                public_keys: Some(vec![public_key_entry.unwrap()]),
+                services: None,
+            },
+            signing_key,
+        )
+    };
```
Comment (sgreenbury) [inline trustchain-ion/src/create.rs:54]: Check and consider refactor before merge

Comment (sgreenbury) [reply trustchain-ion/src/create.rs]: Renamed to `generated_signing_key` to improve clarity.

**Review by thobson88 — APPROVED** [review]
LGTM

**Pull request merged**